pkg/cluster/describe.go

```diff
@@ -272,3 +275,48 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 
 	return nil
 }
+
+// findHyperShiftMgmtSvcClusters returns the name of a HyperShift cluster's management and service clusters.
+// It essentially ignores error as these endpoint is behind specific permissions by returning empty strings when any
+// errors are encountered, which results in them not being printed in the output.
+func findHyperShiftMgmtSvcClusters(conn *sdk.Connection, cluster *cmv1.Cluster) (string, string) {
+	if !cluster.Hypershift().Enabled() {
+		return "", ""
+	}
+
+	hypershiftResp, err := conn.ClustersMgmt().V1().Clusters().
+		Cluster(cluster.ID()).
+		Hypershift().
+		Get().
+		Send()
+	if err != nil {
+		return "", ""
+	}
+
+	mgmtClusterName := hypershiftResp.Body().ManagementCluster()
+	fmMgmtResp, err := conn.OSDFleetMgmt().V1().ManagementClusters().
+		List().
+		Parameter("name", mgmtClusterName).
+		Send()
+	if err != nil {
+		return mgmtClusterName, ""
+	}
+
+	// href is in the format /api/osd_fleet_mgmt/v1/service_clusters/${id}
+	// and we are using it because ocm-sdk-go currently does not export the id directly
+	if kind := fmMgmtResp.Items().Get(0).Parent().Kind(); kind == "ServiceCluster" {
+		href := fmMgmtResp.Items().Get(0).Parent().Href()
+		_, id := path.Split(href)
+		fmSvcResp, _ := conn.OSDFleetMgmt().V1().ServiceClusters().ServiceCluster(id).Get().Send()
+		svcClusterId := fmSvcResp.Body().ClusterManagementReference().ClusterId()
+
+		svcClusterResp, err := conn.ClustersMgmt().V1().Clusters().Cluster(svcClusterId).Get().Send()
+		if err != nil {
+			return mgmtClusterName, ""
+		}
+		return mgmtClusterName, svcClusterResp.Body().Name()
```

Comment:
Why are you looking up the cluster ID if you're returning the name? That should be available in the MC payload already:
```
if kind := fmMgmtResp.Items().Get(0).Parent().Kind(); kind == "ServiceCluster" {
  svcClusterName := fmMgmtResp.Items().Get(0).Parent().Name()
}
```

Reply:
okay, just saw the the SDK doesn't expose the SC name, but it's part of the payload. Can you add this to the corresponding model file? https://github.com/openshift-online/ocm-api-model/blob/main/model/osd_fleet_mgmt/v1/management_cluster_parent_type.model

Reply:
Name isn't exported by ocm-sdk-go :( It's returned when I hit the API via `ocm get` though

Reply:
Added to the model file in https://github.com/openshift-online/ocm-api-model/pull/690 and verified it worked - but now this PR should not be merged until that one is + a corresponding bump in ocm-sdk-go + a dependency bump in ocm-cli. So I am marking this PR as WIP.

Reply:
There's now a new SDK version with support for service/management cluster names. Can you update this?

Reply:
Done! Thanks for pushing the SDK update forward!